pkg/mimirtool/commands/alerts.go

```diff
@@ -147,6 +147,7 @@ func (a *AlertCommand) Register(app *kingpin.Application, envVars EnvVarNames) {
 	alertCmd.Flag("id", "Mimir tenant id, alternatively set "+envVars.TenantID+".").Envar(envVars.TenantID).Required().StringVar(&a.ClientConfig.ID)
 	alertCmd.Flag("user", fmt.Sprintf("API user to use when contacting Grafana Mimir, alternatively set %s. If empty, %s will be used instead.", envVars.APIUser, envVars.TenantID)).Default("").Envar(envVars.APIUser).StringVar(&a.ClientConfig.User)
 	alertCmd.Flag("key", "API key to use when contacting Grafana Mimir; alternatively, set "+envVars.APIKey+".").Default("").Envar(envVars.APIKey).StringVar(&a.ClientConfig.Key)
+	alertCmd.Flag("authToken", "Authentication token for bearer token or JWT auth, alternatively set MIMIR_AUTH_TOKEN.").Default("").Envar("MIMIR_AUTH_TOKEN").StringVar(&a.ClientConfig.AuthToken)
```

Comment:
Can you add this variable to the `EnvVarNames` struct?

Reply:
Thanks, Addressed.